golangci failing for doc/verifiable

level=warning msg="[runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: buildssa@github.com/hyperledger/aries-framework-go/pkg/doc/verifiable [github.com/hyperledger/aries-framework-go/pkg/doc/verifiable.test]"

level=warning msg="[runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/opt/workspace/pkg/doc/verifiable/credential.go:17:2: could not import github.com/VictoriaMetrics/fastcache (/go/pkg/mod/github.com/!victoria!metrics/fastcache@v1.5.4/malloc_mmap.go:24:24: Mmap not declared by package syscall) -: could not load export data: no export data for \"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable\"]"

--- a/pkg/doc/verifiable/cache.go
+++ b/pkg/doc/verifiable/cache.go
@@ -1,0 +1,23 @@
+// +build !js,!wasm
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"time"
+
+	"github.com/VictoriaMetrics/fastcache"
+)
+
+// NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
+func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
+	return &ExpirableSchemaCache{
+		cache:      fastcache.New(size),
+		expiration: expiration,
+	}
+}

--- a/pkg/doc/verifiable/cache_js_wasm.go
+++ b/pkg/doc/verifiable/cache_js_wasm.go
@@ -1,0 +1,18 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import "time"
+
+// NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
+func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
+	// TODO Add cache implementation for VC wasm https://github.com/hyperledger/aries-framework-go/issues/1009
+	return &ExpirableSchemaCache{
+		cache:      nil,
+		expiration: expiration,
+	}
+}

--- a/pkg/doc/verifiable/credential.go
+++ b/pkg/doc/verifiable/credential.go
@@ -14,7 +14,6 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/VictoriaMetrics/fastcache"
 	"github.com/xeipuuv/gojsonschema"
 
 	"github.com/hyperledger/aries-framework-go/pkg/common/log"
@@ -223,18 +222,19 @@ type SchemaCache interface {
 	Get(k string) ([]byte, bool)
 }
 
-// ExpirableSchemaCache is an implementation of SchemaCache based fastcache.Cache with expirable elements.
-type ExpirableSchemaCache struct {
-	cache      *fastcache.Cache
-	expiration time.Duration
+// cache defines a cache interface
+type cache interface {
+	Set(k, v []byte)
+
+	HasGet(dst, k []byte) ([]byte, bool)
+
+	Del(k []byte)
 }
 
-// NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
-func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
-	return &ExpirableSchemaCache{
-		cache:      fastcache.New(size),
-		expiration: expiration,
-	}
+// ExpirableSchemaCache is an implementation of SchemaCache based fastcache.Cache with expirable elements.
+type ExpirableSchemaCache struct {
+	cache      cache
+	expiration time.Duration
 }
 
 // CredentialSchemaLoader defines expirable cache.

